src/pymodaq_plugins_newport/daq_move_plugins/daq_move_Newport_XPS_Q8.py

```diff
@@ -0,0 +1,294 @@
+from pymodaq.control_modules.move_utility_classes import DAQ_Move_base, comon_parameters_fun, main, DataActuatorType,\
+    DataActuator  # common set of parameters for all actuators
+from pymodaq.utils.daq_utils import ThreadCommand # object used to send info back to the main thread
+from pymodaq.utils.parameter import Parameter
+from qtpy.QtCore import QThread
+from pymodaq_plugins_newport.hardware import XPS_Q8_drivers 
+import sys
+
+from time import perf_counter_ns
+
+class XPSPythonWrapper():
```

Comment:
put this wrapper in the hardware folder which is dedicated for that and keep here only the plugin features

Reply:
[476e677](https://github.com/PyMoDAQ/pymodaq_plugins_newport/pull/6/commits/476e677c9edae544c4f6c77cb10a9a425fe91602) moved to new file xps_q8_simplified.py so as to not modify the driver provided by the constructor (XPS_Q8_drivers.py)

Reply:
Sorry for the messy commit, most of the changes in the daq_move file are from my linter